modules/common/src/middleware/elapsed-time.middleware.ts

```diff
@@ -23,20 +23,17 @@ export class ElapsedTimeMiddleware implements NestMiddleware {
         }
 
         const startHrTime: [number, number] = process.hrtime();
-        const requestIdentifier: string = `${req.method} ${req.originalUrl}`;
         res.on('finish', (): void => {
             const elapsedHrTime: [number, number] = process.hrtime(startHrTime);
             const elapsedTimeInMs: number = elapsedHrTime[0] * 1e3 + elapsedHrTime[1] / 1e6;
             this.logger.log({
                 statusCode: `${res.statusCode}`,
-                request: requestIdentifier,
                 elapsedTime: `${elapsedTimeInMs}`,
                 message: 'Route finished',
             });
         });
 
         this.logger.log({
-            request: requestIdentifier,
```

Comment:
we could maybe still print the `req.originalUrl` in here, or if it would be possible to print it as part of `httpRequest` it would be better... now we only have URL with an IP and protocol which is not ideal for search...

Reply:
Ok, so easiest solution here is to have request middleware be first middleware in chain, so it always gets saved first

Reply:
> sadly we do not have data to which tld request was made

it cannot be taken from the request?

Reply:
I checked the whole request object, it looks like it is lost either after fastly or router